docs/source/release/v6.5.0/Diffraction/Powder/New_features/34193 Absolute Normalization.txt

```diff
@@ -0,0 +1 @@
+- Add a switch to allow an absolute normalization to be performed during by the ISIS powder diffraction scripts when running on the POLARIS instrument. As part of this the setting "crystal_density" has been removed from the SampleDetails class. The density of the material can now be set instead by supplying two of "number_density", "number_density_effective" and "packing_fraction"
```

Comment:
```
during by the ISIS powder diffraction ...
```
Just reword slightly. Also should these release note files be .rst??? Maybe it doesn't matter!